store/v2/root/builder.go

```diff
@@ -25,6 +25,8 @@ type Builder interface {
 	RegisterKey(string)
 	// Get returns the Store.  Build should be called before calling Get or the result will be nil.
 	Get() store.RootStore
+	// GetStoreKeys returns the store keys that have been registered.
```

Comment:
Is this needed somewhere? Given we have this on the module manager already: https://github.com/cosmos/cosmos-sdk/blame/main/runtime/v2/manager.go#L87-L101